From ea435ed765f881e56017ebf47edc33cb91602970 Mon Sep 17 00:00:00 2001
From: Daniel Borkmann <dborkman@redhat.com>
Date: Fri, 14 Nov 2014 12:38:47 -0500
Subject: [net] sctp: replace seq_printf with seq_puts

Message-id: <eb550898659e6b7bb818606cbda09dcb3e4b7519.1415968455.git.dborkman@redhat.com>
Patchwork-id: 100051
O-Subject: [RHEL7 PATCH net 2/2] sctp: replace seq_printf with seq_puts
Bugzilla: 1164214
RH-Acked-by: Neil Horman <nhorman@redhat.com>
RH-Acked-by: Marcelo Leitner <mleitner@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1164214
Upstream Status: afb6befce60e3a8b09f7633b9e34e702a72adc0e (net-next)
Brew: https://brewweb.devel.redhat.com/taskinfo?taskID=8246153
Tested: verified on beaker

Upstream Commit:
commit afb6befce60e3a8b09f7633b9e34e702a72adc0e
Author: Michele Baldessari <michele@acksyn.org>
Date:   Thu Oct 30 10:29:16 2014 +0100

    sctp: replace seq_printf with seq_puts

    Fixes checkpatch warning:
    "WARNING: Prefer seq_puts to seq_printf"

    Signed-off-by: Michele Baldessari <michele@acksyn.org>
    Acked-by: Neil Horman <nhorman@tuxdriver.com>
    Signed-off-by: David S. Miller <davem@davemloft.net>

Signed-off-by: Daniel Borkmann <dborkman@redhat.com>
Signed-off-by: Jiri Benc <jbenc@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/net/sctp/proc.c b/net/sctp/proc.c
index 790352f..1ffa01b 100644
--- a/net/sctp/proc.c
+++ b/net/sctp/proc.c
@@ -491,14 +491,14 @@ static int sctp_remaddr_seq_show(struct seq_file *seq, void *v)
     * Note: We don't have a way to tally this at the moment
     * so lets just leave it as zero for the moment
     */
-   seq_printf(seq, "0 ");
+   seq_puts(seq, "0 ");
 
    /*
     * remote address start time (START).  This is also not
     * currently implemented, but we can record it with a
     * jiffies marker in a subsequent patch
     */
-   seq_printf(seq, "0 ");
+   seq_puts(seq, "0 ");
 
    /*
     * The current state of this destination. I.e.
-- 
1.7.1